python/nav/config.py

```diff
@@ -52,6 +52,16 @@
     ] + CONFIG_LOCATIONS
 
 
+def list_config_files_from_dir(dirname):
```

Comment:
This could have a unit test: test the conditional by creating files you want included and excluded, run the function and check that the resulting list contains what you expect.